include/bgfx/defines.h

```diff
@@ -522,6 +522,7 @@
 #define BGFX_PCI_ID_AMD                           UINT16_C(0x1002) //!< AMD adapter.
 #define BGFX_PCI_ID_INTEL                         UINT16_C(0x8086) //!< Intel adapter.
 #define BGFX_PCI_ID_NVIDIA                        UINT16_C(0x10de) //!< nVidia adapter.
+#define BGFX_PCI_ID_MICROSOFT                     UINT16_C(0x1414) //!< Microsoft adapter
```

Comment:
define.h is generated from here:
https://github.com/bkaradzic/bgfx/blob/75fe88fe3afb048ac6f775d9991f67a406e1f4a5/scripts/bgfx.idl#L424-L430